RockLib.UniversalMemberAccessor.Tests/TypeCreator/Create.cs

```diff
@@ -100,6 +100,7 @@ internal static string RandomName()
 
             return new string(new[]
             {
+#pragma warning disable CA5394
```

Comment:
I'm thinking we should start using `RandomNumberGenerator` instead of `Random`. All of these calls should be replaceable with `RandomNumberGenerator.GetInt32(...)` and then we can remove the `Random` object instance along with the `#pragma`.

Reply:
@JasonBock it looks like NET48 doesn't support `.GetInt32()`, so do you think this is still worth implementing in this case for these tests?
[.NET Framework 4.8 RandomNumberGenerator docs](https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.randomnumbergenerator?view=netframework-4.8)

Reply:
Crud. I thought it was there. Oh well.